helpers/nested.js

```diff
@@ -0,0 +1,42 @@
+const flatten = function (data) {
```

Comment:
@therebelrobot I used the 'update branch' button at the bottom of the PR and the changes became a lot more reasonable.